target/linux/ipq806x/image/Makefile

```diff
@@ -294,4 +294,21 @@ define Device/AP-DK04.1-C1
 endef
 TARGET_DEVICES += AP-DK04.1-C1
 
+define Device/openmesh_a42
+	$(call Device/FitImageLzma)
+	FILESYSTEMS := squashfs
+	DEVICE_DTS := qcom-ipq4019-a42
+	KERNEL_LOADADDR := 0x80208000
+	BLOCKSIZE := 64k
+	SUPPORTED_DEVICES := openmesh,a42
+	DEVICE_TITLE := OpenMesh A42
+	KERNEL = kernel-bin | lzma | fit lzma $$(DTS_DIR)/$$(DEVICE_DTS).dtb | pad-to 64k
```

Comment:
Please use `pad-to $$$$(BLOCKSIZE)`

And I finally get why it is required.  the rootfs is padded on it's own and doesn't fix an odd padding of the kernel, as it is done with the usual kernel+rootfs images.